lib/formatter/csv/subject.rb

```diff
@@ -39,9 +39,8 @@ def workflow_ids
       end
 
       def locations
-        subject_locs = subject.locations.sort_by { |loc| loc.metadata["index"] }
```

Comment:
Your refactor is cleaner, but to make this work as-is it would have only needed to have this`sort_by` chopped off, since the `locations.loaded?` is the relevant question, right?